TABLE.md

```diff
@@ -0,0 +1,25 @@
+|                           |                                                                           |                                                                                                                              | - get: function to get the member.`func(instance)` return the field value                                                    | - key: field key                      |
+|                           |                                                                           |                                                                                                                              | - set: function to get the member. `func(instance, value)` update field value and return the value                           |                                       |
+| *return*                  | replace method or class or undefined                                      | undefined                                                                                                                    | undefined                                                                                                                    | undefined                             |
+| **decorate**              |                                                                           |                                                                                                                              |                                                                                                                              |                                       |    
+| *class*                   |                                  X                                        |                                                             X                                                                |                                                             -                                                                |                   -                   |
+| *public method*           |                                  X                                        |                                                             X                                                                |                                                             -                                                                |                   -                   |
+| *public accessor*         |                      X (applies individually)                             |                                                 X (applies individually)                                                     |                                                             -                                                                |                   -                   |
```

Comment:
I'm not sure what value it adds to have "applies individually" to be listed per table entry; there's simply no coalescing in this proposal. This can be explained outside of the table.

Reply:
The psychology about data visualization is complex. There are people who feel overwhelmed with tables and prefer a textual model. Others, like me, love the synthetic versions based on tables that describe in very few lines the behavior of functions, methods or -now- decorators.

Please, ignore this proposal if you believe that it can increase any risk when presenting then new promises.